### PR TITLE
Doc for exponential and examples

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/boost/simd/function/correct_fma.hpp [hunk 1]
```diff
@@ -57,7 +57,8 @@ namespace boost { namespace simd
     - @c correct_fma is never used internally in boost.simd.
 
     @par Decorators
-      std_ for floating entries calls std::fma but does not guarantee performances...
+
+      - std_ for floating entries calls std::fma but does not guarantee performances...
```
Comment (jfalcou) [inline include/boost/simd/function/correct_fma.hpp]: a single . is enough

File: include/boost/simd/function/nthroot.hpp [hunk 2]
```diff
@@ -16,38 +16,35 @@ namespace boost { namespace simd
...
-    is similar to:
-
-    @code
-    T r = n >= 0 ? pow(x, rec(tofloat(n))) : Nan;
-    @endcode
 
     @par Note:
-    nthroot is more expansive than pow(x, rec(tofloat(n))) because
+    nthroot is slower than `pow(x, rec(tofloat(n))`) because
     it takes care of some limits issues that @ref pow does not mind of.
```
Comment (jfalcou) [inline include/boost/simd/function/nthroot.hpp]: "that @ref pow ignores."

File: include/boost/simd/function/nthroot.hpp [hunk 3]
```diff
@@ -16,38 +16,35 @@ namespace boost { namespace simd
...
 
     See if it suits you better or use raw_ decorator for intermediate solution.
 
+    @par Decorators
+
+      - raw_ provides speed but is undefined for limitings values
```
Comment (jfalcou) [inline include/boost/simd/function/nthroot.hpp]: "provides increased speed"

File: include/boost/simd/function/correct_fma.hpp (hunk 1 above)
Comment (jtlap) [reply include/boost/simd/function/correct_fma.hpp]: it seems to me that correct_fma.hpp is a spurious fule. I will verify

File: include/boost/simd/function/nthroot.hpp (hunk 2 above)
Comment (jtlap) [reply include/boost/simd/function/nthroot.hpp]: ok

File: include/boost/simd/function/nthroot.hpp (hunk 3 above)
Comment (jtlap) [reply include/boost/simd/function/nthroot.hpp]: ok

File: include/boost/simd/function/correct_fma.hpp (hunk 1 above)
Comment (jtlap) [reply include/boost/simd/function/correct_fma.hpp]: Yes the only coorect_fma.hpp that must remain are those containing implementation of pedantic_(fma)

File: bench/function/scalar/correct_fma/regular.double.m10_10_m10_10_m10_10.cpp [hunk 4]
```diff
@@ -1,21 +0,0 @@
-// -------------------------------------------------------------------------------------------------
```
Comment (jfalcou) [inline bench/function/scalar/correct_fma/regular.double.m10_10_m10_10_m10_10.cpp:1]: You forgot to remove those from the bench cmake

**Comment by jtlap** [comment]
still travis timeout

**Pull request merged**